applications/app/controllers/ShortUrlsCampaignController.scala

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import common.Logging
+import common.`package`._
+import conf.LiveContentApi
+import model.Cached
+import play.api.mvc.{RequestHeader, Action, Controller}
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+object ShortUrlsCampaignController extends Controller with Logging {
+
+  val campaigns = Map (
+    "iw" -> "twt_ipd",
+    "wc" -> "twt_wc",
+    "tf" -> "twt_fd",
+    "fb" -> "fb_gu",
+    "fo" -> "fb_ot",
+    "us" -> "fb_us",
+    "au" -> "soc_567",
+    "tw" -> "twt_gu",
+    "at" -> "twt_atn",
+    "sfb"-> "share_btn_fb",
+    "ip" -> "twt_iph",
+    "stw" -> "share_btn_tw",
+    "swa" -> "share_btn_wa",
+    "em" -> "email",
+    "sgp" -> "share_btn_gp",
+    "sbl" -> "share_btn_link"
+  )
+
+  def redirectShortUrl(shortUrl: String) = Action.async { implicit request =>
+    log.info(s"Redirecting short url $shortUrl")
+    redirectUrl(shortUrl, request.queryString)
+  }
+
+  private def redirectUrl(shortUrl: String, queryString: Map[String, Seq[String]])(implicit request: RequestHeader) = {
+    LiveContentApi.getResponse(LiveContentApi.item(shortUrl)).map { response =>
+      response.content.map(_.id).map { id =>
+        Redirect(s"/$id", queryString)
```

Comment:
I'm pretty sure this will still remain on `gu.com`, I think you need an absolute url here, not a relative one (maybe the `weburl` field).